docs/operating-scylla/admin-tools/scylla-types.rst

```diff
@@ -57,17 +57,24 @@ of the types on the command line must be the same as the order in the compound).
 
 .. code-block:: console
 
-   scylla types print --prefix-compound -t TimeUUIDType -t Int32Type 0010d00819896f6b11ea00000000001c571b000400000010
+   scylla types deserialize --prefix-compound -t TimeUUIDType -t Int32Type 0010d00819896f6b11ea00000000001c571b000400000010
 
 
 .. _scylla-types-operations:
 
 Supported Operations
 ^^^^^^^^^^^^^^^^^^^^^^^
-* ``print`` - Deserializes and prints the provided value in a human-readable form. Required arguments: 1 or more serialized values.
+* ``serialize`` - Serialize the value and print it in a hex encoded form. Required arguments: 1 value in human-readable form. To avoid problems around special symbols, separate values with ``--`` from the rest of the arguments.
```

Comment:
```suggestion
* ``serialize`` - Serializes the value and prints it in a hex-encoded form. Required arguments: 1 value in a human-readable form. To avoid problems around special symbols, separate the values from the rest of the arguments with ``--``.
```